network/openr/Makefile

```diff
@@ -0,0 +1,52 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=openr
+PKG_SOURCE_DATE:=2019-05-06
+PKG_SOURCE_VERSION:=9437bbaff3e463a56f592c60d33e4fc33e2e27f9
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_SOURCE_DATE).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/facebook/openr/tar.gz/$(PKG_SOURCE_VERSION)?
+PKG_HASH:=15acb76b5867e247f9d15342b7cad3f03028f8b8e19ae515e0f5cc5f1a9557b5
+PKG_BUILD_DIR:=$(BUILD_DIR)/openr-$(PKG_SOURCE_VERSION)
+
+PKG_BUILD_DEPENDS:=fbthrift/host
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+PKG_BUILD_PARALLEL:=1
+CMAKE_BINARY_SUBDIR:=build
+CMAKE_OPTIONS:= \
+	-DCMAKE_POSITION_INDEPENDENT_CODE=True \
+	-DBUILD_TESTS=OFF \
+	-DBUILD_SHARED_LIBS=ON \
+	-DCMAKE_EXE_LINKER_FLAGS=-latomic \
+	-DTHRIFT1="$(STAGING_DIR_HOST)/bin/thrift1"
+TARGET_LDFLAGS:=-lboost_program_options -lboost_regex -lboost_context
+CMAKE_INSTALL:=1
+
+define Package/openr
+	SECTION:=network
+	CATEGORY:=Network
+	TITLE:=OpenR: Distributed platform for building autonomic network functions.
+	DEPENDS:=+libstdcpp +fbthrift +libfolly +libsigar +libzmq-nc \
```

Comment:
Some of these like libstdcpp can be eliminated as they are implied by other packages.